Gemfile.lock

```diff
@@ -277,34 +272,34 @@ GEM
       activerecord (>= 5.0)
     psych (3.1.0)
     public_suffix (4.0.1)
-    puma (4.2.0)
+    puma (4.3.1)
       nio4r (~> 2.0)
-    rack (2.0.7)
+    rack (2.0.8)
     rack-cors (0.4.1)
     rack-proxy (0.6.5)
       rack
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
-    rails (6.0.0)
-      actioncable (= 6.0.0)
-      actionmailbox (= 6.0.0)
-      actionmailer (= 6.0.0)
-      actionpack (= 6.0.0)
-      actiontext (= 6.0.0)
-      actionview (= 6.0.0)
-      activejob (= 6.0.0)
-      activemodel (= 6.0.0)
-      activerecord (= 6.0.0)
-      activestorage (= 6.0.0)
-      activesupport (= 6.0.0)
+    rails (6.0.2.1)
+      actioncable (= 6.0.2.1)
+      actionmailbox (= 6.0.2.1)
+      actionmailer (= 6.0.2.1)
+      actionpack (= 6.0.2.1)
+      actiontext (= 6.0.2.1)
+      actionview (= 6.0.2.1)
+      activejob (= 6.0.2.1)
+      activemodel (= 6.0.2.1)
+      activerecord (= 6.0.2.1)
+      activestorage (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
       bundler (>= 1.3.0)
-      railties (= 6.0.0)
+      railties (= 6.0.2.1)
       sprockets-rails (>= 2.0.0)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
-    rails-html-sanitizer (1.2.0)
-      loofah (~> 2.2, >= 2.2.2)
+    rails-html-sanitizer (1.3.0)
```

Comment:
### [rails-html-sanitizer](https://github.com/rails/rails-html-sanitizer)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.2.0 → 1.3.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **4** commits. See the full changes on [the compare page](https://github.com/rails/rails-html-sanitizer/compare/v1.2.0...v1.3.0).

These are the individual commits:
- (43a87f5) [Match Loofah's API changes.](https://github.com/rails/rails-html-sanitizer/commit/43a87f538af42dd11cd72fddacbba2519d1e5c78)
- (845da04) [Add project metadata to the gemspec](https://github.com/rails/rails-html-sanitizer/commit/845da04cad37885aadc4495c8e7e476de54aa803)
- (65b9f88) [Merge pull request #102 from orien/gem-metadata](https://github.com/rails/rails-html-sanitizer/commit/65b9f880f23abe9c7f2719da58bdf613f5f5c307)
- (51dc564) [v1.3.0](https://github.com/rails/rails-html-sanitizer/commit/51dc564c6509201070f72456bb2c13f87bb373d6)


</details>